pkg/kernel/networkservice/connectioncontextkernel/ipcontext/ipaddress/common.go

```diff
@@ -96,6 +96,11 @@ func create(ctx context.Context, conn *networkservice.Connection, isClient bool)
 		if err := netlink.AddrSubscribeAt(targetNetNS, ch, done); err != nil {
 			return errors.Wrapf(err, "failed to subscribe for interface address updates")
 		}
+		defer func() {
+			// `ch` should be fully read after the `done` close to prevent goroutine leak in `netlink.AddrSubscribeAt`
```

Comment:
`defer close(done)` happens at line 95 after this iteration, shouldn't we change it ?

Reply:
Yes, thank you.

Reply:
Fixed.